utils/rpms/daos.spec

```diff
@@ -298,6 +298,7 @@ This is the package that bridges the difference between the MOFED openmpi
       --config=force         \
       --no-rpath             \
       USE_INSTALLED=all      \
+      STATIC_FUSE=0          \
```

Comment:
I need to figure out a better way but for now, we'll just use the upstream version of fuse for these builds.